refl1d/probe.py

```diff
@@ -1,7 +1,4 @@
-# coding=utf-8
-# This program is in the public domain
```

Comment:
This attribution and others like it can be removed... we have already provided attribution for Paul in the docs.